company_multi_logo/README.rst

```diff
@@ -0,0 +1,56 @@
+Company Multiple Logo
+=====================
+
+This module allows to display different logos in reports for a single company.
+For each partner, a logo can be selected to be displayed in the reports for this partner.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+ * Install the module, and then modify the Qweb template report.external_layout_header to display the logo.
```

Comment:
By default, the field 'logo' of the company is displayed in the header. You need to edit your header so that a logo from the field 'logo_ids' (Many2many field) is displayed. How it is displayed (what width, length, position) should be the company's choice. This is why I do not edit the header in this module.